iaso/tests/api/test_mobile_orgunits.py

```diff
@@ -211,3 +226,66 @@ def test_boundingbox_with_user_limit(self):
         # set limit on user
         bbox = j["results"][0]
         self.assertEqual(bbox, {"east": -5.024, "northern": -4.024, "south": -3.976, "west": -4.976})
+
+    @time_machine.travel("2023-10-26T09:00:00.000Z", tick=False)
+    def test_reference_instances(self):
+        # Instance 1.
+        form1 = Form.objects.create(name="Form 1")
+        form_version1 = FormVersion.objects.create(form=form1, version_id=1)
+        instance1 = Instance.objects.create(
+            form=form1, org_unit=self.raditz, json={"key": "foo"}, form_version=form_version1
+        )
+        # Instance 2.
+        form2 = Form.objects.create(name="Form 2")
+        form_version2 = FormVersion.objects.create(form=form2, version_id=5)
+        instance2 = Instance.objects.create(
+            form=form2, org_unit=self.raditz, json={"key": "bar"}, form_version=form_version2
+        )
+        # Mark instances as reference instances.
+        OrgUnitReferenceInstance.objects.create(org_unit=self.raditz, instance=instance1, form=form1)
+        OrgUnitReferenceInstance.objects.create(org_unit=self.raditz, instance=instance2, form=form2)
+
+        self.client.force_authenticate(self.user)
+
+        params = {APP_ID: BASE_APP_ID, "limit": 10}
+
+        # Fetch OrgUnit by ID.
+        response1 = self.client.get(f"{BASE_URL}{self.raditz.pk}/reference_instances/", data=params)
+        self.assertJSONResponse(response1, 200)
+
+        # Fetch OrgUnit by UUID.
+        response2 = self.client.get(f"{BASE_URL}{self.raditz.uuid}/reference_instances/", data=params)
+        self.assertJSONResponse(response2, 200)
+
+        self.assertEqual(response1.data, response2.data)
+        self.assertEqual(
+            response1.data,
+            {
+                "count": 2,
+                "orgUnits": [
```

Comment:
Shouldn't it be `"instances"` rather than `"orgUnits"`?

Reply:
Yup, will fix.

Reply:
Fixed in https://github.com/BLSQ/iaso/pull/896/commits/7639ce7ff705399a41b59562d42e357bcfb1bc34.